Script/AtomicNET/AtomicNET/Scene/CSComponentCore.cs

```diff
@@ -332,6 +332,21 @@ void HandleComponentLoad(uint eventType, ScriptVariantMap eventData)
 
         }
 
+        void HandleComponentAssemblyReference(uint eventType, ScriptVariantMap eventData)
+        {
+#if ATOMIC_DESKTOP || ATOMIC_ANDROID
+            string assemblyPath = eventData["AssemblyPath"];
```

Comment:
Kept it because the others were Android included. If there are any additional dlls in an Android project, would also be needed there?